task-10/js/app.js

```diff
@@ -0,0 +1,105 @@
+// Section 1
+
+console.log(1 + 2);
+console.log("apple" + "orange");
+console.log(1 + 2 + "apple");
+console.log("apple" + 1 + 2);
+console.log(1 + true);
+console.log(0 == false);
+console.log(1 === true);
+console.log(2 == "2");
+
+// Section 2
+
+// 1
+
+cricketPlayers = ["Ashok", "Harish", "Dinesh", "Maamathi", "Kavya", "Ranjith", "Charan", "Hareesh", "Mathi", "Thendral", "Pavithra"]
+
+console.log(cricketPlayers);
+
+// 2
+
+console.log(cricketPlayers.shift())
+
+// 3
+
+console.log(cricketPlayers.length)
+
+// 4
+
+cricketPlayers.push("Ashok")
+console.log(cricketPlayers)
+
+// 5
+
+cricketPlayers.sort(function (a, b) {
+    return a.toLowerCase() - b.toLowerCase();
+});
+console.log(cricketPlayers);
+
+// 6
+
+cricketPlayersWithJerseyNumber = cricketPlayers.map((value) => value + " - " + Math.round(Math.random() * 100))
+console.log(cricketPlayersWithJerseyNumber);
+
+// 7
+
+cricketPlayersUpperCase = cricketPlayersWithJerseyNumber.map(value => value.toUpperCase())
```

Comment:
1. missing declaration;
2 . I believe you should use the players instead the playersWithJerseyNumbers